src/pages/registration/resource_type_tab/sub_type_forms/degree_types/PhdForm.tsx

```diff
@@ -68,14 +69,21 @@ export const PhdForm = () => {
                     gap: '0.25rem 1rem',
                     mb: '0.5rem',
                   }}>
-                  <Field name={`${ResourceFieldNames.PublicationInstanceRelated}[${index}].text`}>
-                    {({ field }: FieldProps<string>) => (
+                  <Field
+                    name={`${ResourceFieldNames.PublicationInstanceRelated}[${index}].text`}
+                    validate={(value: string) =>
```

Comment:
Skulle ikke dette feltet blitt validert av modellen vi har i formik som alle andre typer?
Altså ved å legges til i `degreePublicationInstance` i `referenceValidation.ts`. Ellers blir det fort vanskelig å se hvor alle feilmeldinger trigges fra da

Reply:
Usikker på om formik submit tar hensyn til sånne inline-valideringer ved submit også, eller om det bare er for visning til bruker 🤔